include/grpcpp/security/tls_credentials_options.h

```diff
@@ -0,0 +1,293 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GRPCPP_SECURITY_TLS_CREDENTIALS_OPTIONS_H
+#define GRPCPP_SECURITY_TLS_CREDENTIALS_OPTIONS_H
+
+#include <memory>
+#include <vector>
+
+#include <grpc/grpc_security.h>
+#include <grpc/support/log.h>
+#include <grpcpp/support/config.h>
+
+namespace grpc_impl {
+namespace experimental {
+
+/** TLS key materials config, wrapper for grpc_tls_key_materials_config. It is
+ * used for experimental purposes for now and subject to change. **/
+class TlsKeyMaterialsConfig {
+ public:
+  struct PemKeyCertPair {
+    grpc::string private_key;
+    grpc::string cert_chain;
+  };
+
+  /** Getters for member fields. **/
+  const grpc::string pem_root_certs() const { return pem_root_certs_; }
```

Comment:
nit: is a reason to use c style function name? Why not use PemRootCerts()?
same question for rest of this file.

Reply:
I was using the convention specified in the last paragraph of the "Function names" section from the C++ style guide: http://go/cstyle#Function_Names . My understanding was that getter and setter functions should be named like variables, but I'm happy to change it if you prefer the style you suggested.